src/3p-frame.js

```diff
@@ -157,8 +161,8 @@ export function getIframe(parentWindow, parentElement, opt_type, opt_context) {
     // Chrome does not reflect the iframe readystate.
     this.readyState = 'complete';
   };
-  iframe.setAttribute(
-      'data-amp-3p-sentinel', attributes._context.amp3pSentinel);
+  iframe.setAttribute('data-amp-3p-sentinel', attributes._context[
```

Comment:
The sentinel refactor one does, but I was under the impression this one should go through first. Hence, this one still has it.